tests/informatics/test_predictors.py

```diff
@@ -231,21 +231,6 @@ def test_molecule_featurizer(molecule_featurizer):
     }
 
 
-def test_molecule_featurizer_deprecation():
```

Comment:
Why can we 🪓 these tests now? (and the ones in table config)

Reply:
they're testing the different permutations of how arguments can be specified (both/neither/new arg/old arg). But that logic is now consolidated and tested in one place.